lib/registration.pm

```diff
@@ -60,6 +61,7 @@ our %SLE15_MODULES = (
     contm     => 'Containers',
     pcm       => 'Public-Cloud',
     sapapp    => 'SAP-Applications',
+    python2   => 'Python2',
```

Comment:
I'm not sure if we need to add python2 here, anyway it looks like this won't cause wrong.

Reply:
I have several reasons for this:
1. This is the  abbreviation of sle15 modules, we should mark it at here, in case, someone use other abbreviation to stand for Python2 module, it will mess up for the module's name.
2. This directory will be used at 'sub is_module': Method to determine if a short name references a module based on what's defined. User can check if the abbreviation is the sle15 module or not.
3. I think it is common method to put all the abbreviation at one directory, it is easy for other developer to use and maintain all the abbreviation.
So, I think this is necessary to add the abbreviation of 'Python2' module at here.